commands/utils.py

```diff
@@ -130,6 +130,8 @@ def setup_templates():
             "public-manifest": templateEnv.get_template("svc/manifest-public.yml"),
             "backend-manifest": templateEnv.get_template("svc/manifest-backend.yml"),
             "s3-policy": templateEnv.get_template("addons/svc/s3-policy.yml"),
+            "s3": templateEnv.get_template("addons/svc/s3-policy.yml"),
```

Comment:
These two reference the same template - is this intentional?

Reply:
Yes, I'm  using the contents of `templates["env"]` and `templates["svc"]` to determine if addons should be created at the svc or env level.  

```
storage_type = "s3"

if storage_type in templates["svc"]:
    # create service level template
```

I'm considering refactoring the template logic so that we use a template loader, ditch manual loading in `utils.get_templates()` and then have a dict or yaml file that determines which addons require which files, e.g. 

```
redis:
  env:
    - template: addons/env/redis.yml
    - template: addons/env/addons.parameters.yml
       filename: addons.parameters.yml
s3:
  env:
    - template: addons/env/s3.yml
  svc:
    - template: addons/svc/s3-policy.yml
s3-policy:
  svc:
    - template: addons/svc/s3-policy.yml
```

Reply:
Will be resolved in https://github.com/uktrade/copilot-tools/pull/127.